superset-frontend/src/dashboard/components/HeaderActionsDropdown.jsx

```diff
@@ -217,6 +218,17 @@ class HeaderActionsDropdown extends React.PureComponent {
             onChange={this.changeCss}
           />
         )}
+
+        {!editMode && (
+          <MenuItem
+            onClick={downloadAsImage(
+              '.dashboard',
+              generateFileStem(dashboardTitle),
```

Comment:
instead of needing to call `generateFileStem` and passing it into `downloadAsImage`, I think it's cleaner to pass in a name/description field that `downloadAsImage` then uses to generate the file stem within the function

Reply:
updated